CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+# WildAid O-FISH Web App Contribution Guidelines
+
+There are more details at the [official contribution guidelines](https://wildaid.github.io/contribute/).
+
+In a nutshell:
+Fork the repo
+Clone the repo
+Track changes
+Create a new branch on your fork
+Change and TEST your code
+Commit and push
+Create a pull request
+Wait for a review
+Fix anything the review brought up
+Celebrate when the code gets merged!
```

Comment:
If this is meant to be a bulleted list, then should add `-` to the start of each line

Reply:
@am-MongoDB updated!